source/designers-guide/range-slider-algorithm/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Choosing the right range slider algorithm
+github_link: designers-guide/range-slider-algorithm/index.md
+indexed: true
+---
+
+## Table of contents
+-   [The Problem](#the-problem)
+-   [What is the difference](#what-is-the-difference)
+    -   [Linear](#linear)
+    -   [Log](#log)
+-   [Template Customization](#template-customization)
+
+## The Problem
+
+In Shopware 5, we introduced our new filter system. With this new filter system, there is a range slider which is used as price range slider too.
+
+Imagine one of your categories is full of articles with prices between 4,50 EUR and 70,00 EUR and you have some products around 850,00 EUR. That would result in very high price differences with a very high maximal range. If you start to slide, you will see that the first slide step would be about 40 EUR. Your lower priced products won't be visible anymore. By default, the slider calculates 20 steps between the minimal and the maximal price with a linear function. So basically, the step width is calculated by the highest price divided by the step count.
+
+In 5.0.3, we added the log algorithm for calculation to cover this scenario too!
```

Comment:
In 5.0.3, we added the `log` algorithm for calculation to cover this scenario too.